.github/workflows/report-package-size.yml

```
@@ -0,0 +1,30 @@
+name: Report Package Size
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the application's source code
+        uses: actions/checkout@v2
+
+      - name: Install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://npm.pkg.github.com
+          scope: "@university-of-york"
+          always-auth: true
+
+      - name: Check and report changes in the application's package size
+        uses: preactjs/compressed-size-action@e2bfc4b346d78f9a3c7e8a2e22bcbd285a25fafa
```

Do we want to pin to a specific commit? The [documentation](https://github.com/preactjs/compressed-size-action#usage) suggests:
```suggestion
        uses: preactjs/compressed-size-action@v2
```

---

Commit pinning is recommended in our [guide on using third party actions](https://wiki.york.ac.uk/display/ittechdocs/Use+of+Third+Party+GitHub+Actions) - admittedly I was the one who wrote this guide so happy to start a discussion 🙂

---

Ah, fair enough, I had forgotten about that. Thanks for the reminder!